shared/devices/container.js

```diff
@@ -2,7 +2,7 @@
 import Devices from '.'
 import {List} from 'immutable'
 import {addNewPhone, addNewComputer} from '../actions/login/creators'
-import {compose, lifecycle, mapProps, withState} from 'recompose'
+import {compose, lifecycle, mapProps, withState, withHandlers} from 'recompose'
```

Comment:
not much new here, just a sort change and a handler so we don't need closures in the dumb component